addons/product/report/product_pricelist_report_templates.xml

```diff
@@ -34,31 +36,29 @@
                                 <tr>
                                     <td t-att-class="is_product_tmpl and 'fw-bold' or None">
                                         <a t-if="is_html_type" href="#" class="o_action" t-att-data-model="is_product_tmpl and 'product.template' or 'product.product'" t-att-data-res-id="product['id']">
-                                            <t t-out="product['name']"/>
+                                            <span t-out="product['name']">Acme Widget</span>
```

Comment:
![Acme](https://media2.giphy.com/media/sM1VXgNGTDTc4/giphy.gif?cid=ecf05e474rvq2nu31mp21modmt50rx5ovroj5lpf24ks69v1&ep=v1_gifs_search&rid=giphy.gif&ct=g)

Reply:
Now I have made changes to `Virtual Interior Design` instead of `Acme Widget`.dev has copied the from below line.

Reply:
oh no Acme was fine, i just thought it was funny!

Reply:
don't change it back, no need to undo/redo/re-undo/re-redo for such a small thing